### PR TITLE
Add reduced chisq to remove_background

### PR DESCRIPTION
**Pull request by ericpre**

### CODE REVIEW

File: hyperspy_gui_ipywidgets/tools.py [hunk 1]
```diff
@@ -441,8 +441,10 @@ def remove_background_ipy(obj, **kwargs):
     wdict = {}
     left = ipywidgets.FloatText(disabled=True, description="Left")
     right = ipywidgets.FloatText(disabled=True, description="Right")
+    red_chisq = ipywidgets.FloatText(disabled=True, description="χ²")
```
Comment (francisco-dlp) [inline hyperspy_gui_ipywidgets/tools.py:444]: χ² -> red-χ²

same for traitsui

**Comment by ericpre** [comment]
This PR should be good now!

**Comment by francisco-dlp** [comment]
The tests are not passing, is the skimage constaint in hyperspy's setup.py the issue?

**Comment by ericpre** [comment]
There are several failures, one have been introduced in https://github.com/hyperspy/hyperspy/pull/2307. Others are unrelated.

**Comment by ericpre** [comment]
Tests are now passing with the hyperspy development branch.

**Pull request merged**